docs/AuditListener.png.txt

```diff
@@ -0,0 +1,51 @@
+=======================
+Checkstyle: AuditListener Class Relationship Diagram
+=======================
+
+This text-based Mermaid diagram illustrates the class relationships and
+interface structure of `AuditListener` within Checkstyle.
+
+----------------------------------------
+Mermaid Diagram Code (Paste into https://www.mermaidchart.com/  or https://mermaid.live/edit)
+----------------------------------------
+
+Direct access link:
+https://www.mermaidchart.com/app/projects/39d0649c-6c6e-4135-bf24-dbbe2f13f8a0/
+diagrams/50621826-f5d1-4013-aae7-406b16d969ab/version/v0.1/edit
+
+
+```mermaid
+%%{init: {"theme":"default", "themeVariables": {
```

Comment:
move mermaid content to Auditlistener.png.md file
github will render it itself.